jenkins/bootstrap_test.py

```diff
@@ -903,15 +903,6 @@ def testCIPaths(self):
             self.assertFalse(any(
                 PULL in o for o in (path.a, path.kw)))
 
-    def testNoFinishWhenStartFails(self):
```

Comment:
instead of deleting this test case, change it so it confirms that `finish` is called even if `start` fails?